contextualized/dags/lightning_modules.py

```diff
@@ -409,7 +412,7 @@ def _maybe_update_alpha_rho(self, epoch_dag_loss, dag_params):
         """
         if (
             dag_params.get("use_dynamic_alpha_rho", False)
-            and epoch_dag_loss > dag_params["tol"] * dag_params["h_old"]
+            and epoch_dag_loss > dag_params.get("tol", .25) * dag_params.get("h_old", 0)
```

Comment:
`h_old` is not a parameter and shouldn't be in default args, but this is part of a broader issue with kwarg management and isn't critical here.